src/models/room.ts

```diff
@@ -636,7 +640,7 @@ export class Room extends ReadReceipt<RoomEmittedEvents, RoomEventHandlerMap> {
                     "to be supporting a newer room version we don't know about.",
             );
 
-            const caps = await this.client.getCapabilities(true);
+            const caps = await this.client.fetchCapabilities();
```

Comment:
this would throw here now? instead of returning a default value? Would impact the react-sdk?

Reply:
yeah, I forgot to catch the exception here actually.